src/components/common/styles.js

```diff
@@ -163,64 +188,71 @@ export const StyledError = styled.div`
 
 export const StyledDiv = styled.div`
   width: 30rem;
-  margin: 7rem 0 0 10rem;
+  margin: 24rem 0 0 10rem;
   position: fixed;
   text-align: center;
   font-size: 1rem;
 `;
 
 export const StyledTitle = styled.div`
-  text-align: middle;
+  text-align: left;
   font-size: 3rem;
-  color: white;
+  color: #242741;
   font-weight: bold;
-  margin: 3rem;
+  margin: -1rem;
+`;
+
+export const StyledBody = styled.body`
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  border-bottom: 15rem solid #242741;
+  margin: -15rem 0 0 0;
 `;
 
 /* login page background styled components */
 export const StyledArea = styled.div`
-  background: linear-gradient(0deg, rgba(36, 39, 65, 1) 10rem, rgba(255, 255, 255, 1) 10rem, rgba(36, 39, 65, 1) 40rem);
+  background: repeating-linear-gradient(
+  to bottom, 
+  rgba(255, 255, 255, 1) 35rem,
+  rgba(255, 255, 255, 1) 35rem, 
```

Comment:
is it possible to replace these rgba()'s with hex codes via theme?

Reply:
Yes it should be, there's also some rgba to hex converters online. I'll make them all hex for consistency

Reply:
Actually, problem is hex cannot represent opacity which I need for some of the box shadows on the forms

Reply:
I replaced all of them to hex and place some new ones inside App.js, but there is one rgba that I used for box-shadow opaqueness